open-ce/build_env.py

```diff
@@ -54,12 +54,14 @@ def _run_tests(build_tree, conda_env_files):
     failed_tests = []
     # Run test commands for each conda environment that was generated
     for variant_string, conda_env_file in conda_env_files.items():
-        test_commands = build_tree.get_test_commands(variant_string)
-        if test_commands:
+        test_feedstocks = build_tree.get_test_feedstocks(variant_string)
+        if test_feedstocks:
```

Comment:
I'm assuming there are cases where `build_tree.get_test_feedstocks` doesn't return anything?  And the code is fine skipping that `variant_string` without any messages?

Reply:
Yeah, if there is a case where no tests exist for an entire variant it won't print out the "Running tests" message.